package/smt.spec

```diff
@@ -178,6 +180,7 @@ fi
 
 %preun
 %service_del_preun smt-schema-upgrade.service
+%service_del_preun smt.service
 %service_del_preun smt.target
 
 # no postun service handling, we don't want them to be restarted on upgrade
```

Comment:
What about `%service_del_postun smt.service`? Do we want certificate to be checked automatically after upgrade?